Python/setup.py

```diff
@@ -157,6 +157,8 @@ def finalize_options(self):
                 os.popen('quantlib-config --cflags').read()[:-1].split()
             ql_link_args = \
                 os.popen('quantlib-config --libs').read()[:-1].split()
+            if 'LINKARGS' in os.environ:
+                ql_link_args += os.environ['LINKARGS'].split()
```

Comment:
The variable should probably be `LDFLAGS` (as it is for the autotools).  For consistency, it might also be moved down with `CXXFLAGS` and added to `extra_link_args` instead.